forui/lib/src/widgets/label.dart

```diff
@@ -0,0 +1,466 @@
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      mainAxisSize: MainAxisSize.min,
+      children: [
+        if (label != null)
+          Padding(
+            padding: style.labelPadding,
+            child: DefaultTextStyle(
+              style: stateStyle.labelTextStyle,
+              textHeightBehavior: const TextHeightBehavior(applyHeightToFirstAscent: false),
+              child: label!,
+            ),
+          ),
+        Padding(
+          padding: style.childPadding,
+          child: child,
+        ),
+        if (description != null)
+          Padding(
+            padding: style.descriptionPadding,
+            child: DefaultTextStyle(
+              style: stateStyle.descriptionTextStyle,
+              textHeightBehavior: const TextHeightBehavior(applyHeightToFirstAscent: false),
+              child: description!,
+            ),
+          ),
+        if (error != null && state == FLabelState.error)
+          Padding(
+            padding: style.errorPadding,
+            child: DefaultTextStyle(
+              style: style.formFieldStyle.errorStyle.errorTextStyle,
+              textHeightBehavior: const TextHeightBehavior(applyHeightToFirstAscent: false),
+              child: error!,
+            ),
+          ),
+      ],
+    );
+  }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties
+      ..add(DiagnosticsProperty('style', style))
+      ..add(EnumProperty('state', state));
+  }
+}
+
+/// The [FLabel]'s styles containing [FLabelStyle].
+final class FLabelStyles with Diagnosticable {
+  /// The horizontal label's style.
+  final FLabelStyle horizontal;
+
+  /// The vertical label's style.
+  final FLabelStyle vertical;
+
+  /// Creates a [FLabelStyles].
+  const FLabelStyles({
+    required this.horizontal,
+    required this.vertical,
+  });
+
+  /// Creates a [FLabelStyles] that inherits its properties from the given [style].
+  FLabelStyles.inherit({required FStyle style})
+      : horizontal = FLabelStyle(
+          formFieldStyle: style.formFieldStyle,
+          childPadding: const EdgeInsets.only(right: 8),
+          descriptionPadding: const EdgeInsets.only(top: 2),
+          errorPadding: const EdgeInsets.only(top: 2),
+        ),
+        vertical = FLabelStyle(
+          formFieldStyle: style.formFieldStyle,
+          labelPadding: const EdgeInsets.only(bottom: 5),
+          descriptionPadding: const EdgeInsets.only(top: 5),
+          errorPadding: const EdgeInsets.only(top: 5),
+        );
+
+  /// Returns a copy of this [FLabelStyles] with the given properties replaced.
+  ///
+  /// ```dart
+  /// final style = FLabelStyles(
+  ///   horizontal: ...,
+  ///   vertical: ...,
+  /// );
+  ///
+  /// final copy = style.copyWith(vertical: ...);
+  ///
+  /// print(style.horizontal == copy.horizontal); // true
+  /// print(style.vertical == copy.vertical); // false
+  /// ```
```

Comment:
```suggestion
```

After some consideration, I kinda regret requiring an example for `copyWith(...)`, they don't really seem to add much value.